application/comit_node/src/http_api/problem.rs

```diff
@@ -60,17 +63,31 @@ pub fn serialize(e: serde_json::Error) -> HttpApiProblem {
 pub fn not_yet_implemented(feature: &str) -> HttpApiProblem {
     error!("{} not yet implemented", feature);
     HttpApiProblem::with_title_and_type_from_status(StatusCode::INTERNAL_SERVER_ERROR)
+        .set_title("Feature not yet implemented.")
         .set_detail(format!("{} is not yet implemented! Sorry :(", feature))
 }
 
 pub fn action_already_done(action: Action) -> HttpApiProblem {
     error!("{} action has already been done", action);
-    HttpApiProblem::new("action-already-done").set_status(StatusCode::INTERNAL_SERVER_ERROR)
+    HttpApiProblem::with_title_and_type_from_status(StatusCode::INTERNAL_SERVER_ERROR)
```

Comment:
:clown_face: This shouldn't be a 500, since it is clearly the clients fault. Retrying will not make this error go away :)

410 GONE seems to be very appropriate for this one:

> 410 Gone
> Indicates that the resource requested is no longer available and will not be available again. This should be used when a resource has been intentionally removed and the resource should be purged. Upon receiving a 410 status code, the client should not request the resource in the future. Clients such as search engines should remove the resource from their indices.[43] Most use cases do not require clients and search engines to purge the resource, and a "404 Not Found" may be used instead.